GUI/QtQuick/QtQuickAdapter.cpp

```diff
@@ -1806,3 +1808,28 @@ qtquick_gui::WalletPropertiesVM* QtQuickAdapter::walletProperitesVM() const
 {
    return walletPropertiesVM_.get();
 }
+
+int QtQuickAdapter::exportWallet()
```

Comment:
You should add some wallet identification to those methods (index or walletId).

Reply:
No, you should add a walletId then into the arguments:
`int QtQuickAdapter::changePassword(const QString& walletId, const QString& oldPassword, const QString& newPassword)`